frontend/pages/hubs/[hubUrl].js

```diff
@@ -110,12 +140,15 @@ export default function Hub({
         page: 1,
         token: token,
         urlEnding: newUrlEnding,
+        // TODO: I think this is the primary difference between the applyNewFilters logic
+        // here locally in [hubUrl] and within browse.js -- we should deduplicate
```

Comment:
@positiveimpact added this TODO - I _think_ we can dedupe this `applyNewFilters` logic with what's in `browse` but didn't want to take that on in this PR, so I duplicated the query param persistence here for now.

Reply:
Yep, definitely possible to dedupe atm. I did it like this on purpose incase they deviate further from each other in the future, but it's probably better to just make once function in `BrowseContent`.